lib/hypervisor/hv_kvm/__init__.py

```diff
@@ -1123,6 +1123,12 @@ def _GenerateKVMBlockDevicesOptions(self, up_hvp, kvm_disks,
                           " to prevent shared storage corruption on migration",
                           disk_cache)
         cache_val = ",cache=none"
+      elif aio_mode == constants.HT_KVM_AIO_NATIVE and disk_cache != "none":
+          # TODO: make this a hard error, instead of a silent overwrite
+          logging.warning("KVM: overriding disk_cache setting '%s' with 'none'"
+                          " to prevent QEMU failures in version 2.6+",
+                          disk_cache)
+        cache_val = ",cache=none"
```

Comment:
cache_val indentation level here is incorrect:

Makefile:4691: recipe for target 'man/ganeti-confd.gen' failed
make: *** [man/ganeti-confd.gen] Error 1
make: *** Waiting for unfinished jobs....
Traceback (most recent call last):
  File "/Ganeti/projects/master+scratchpad/ganeti/./autotools/docpp", line 38, in <module>
    from ganeti import query
  File "/tmp/gntbuild.RGa9IUn1/ganeti/query.py", line 77, in <module>
    from ganeti.hypervisor import hv_base
  File "/tmp/gntbuild.RGa9IUn1/ganeti/hypervisor/__init__.py", line 40, in <module>
    from ganeti.hypervisor import hv_kvm
  File "/tmp/gntbuild.RGa9IUn1/ganeti/hypervisor/hv_kvm/__init__.py", line 1131
    cache_val = ",cache=none"
                            ^
IndentationError: unindent does not match any outer indentation level